fastlane/spec/actions_specs/adb_spec.rb

```diff
@@ -49,7 +49,7 @@
         end").runner.execute(:test)
 
         path = "/usr/local/android-sdk/with space/platform-tools/adb".shellescape
-        expect(result).to eq("#{path} test")
+        expect(result).to eq("#{File.expand_path(path)} test")
```

Comment:
I think this might be needed on `path` before the `.shellescape` is called on it.

Reply:
Makes sense, fixed via e301295031b7b84c6efb3e59b283952812a42899.